source/templates.tex

```diff
@@ -3024,7 +3070,10 @@
 }
 \end{codeblock}
 \end{example}
-There is no syntax to form a \grammarterm{template-id}\iref{temp.names}
+An expression designating
+a particular specialization of a conversion function template
+can only be formed with a \grammarterm{splice-expression}.
+There is no analogous syntax to form a \grammarterm{template-id}\iref{temp.names}
```

Comment:
P2996R13 appends " for such a function" to this line

Reply:
... and leaves out the last part of the sentence without showing strike-out.  Fixed.